buildlib/azure-pipelines.yml

```diff
@@ -34,6 +40,18 @@ stages:
     dependsOn: Prepare
     condition: eq(dependencies.Prepare.outputs['Check.Result.Launch'], 'True')
     jobs:
+      - template: jucx/jucx-build.yml
```

Comment:
Should you pass `target` here? Default value is empty, it is a bit confusing.

Reply:
The `target `param is used the `jucx-publish.ym` template, but not with the` jucx-build.yml`.
I would rather leave the default value empty so it will fail, instead of doing something unexpected.